src/screens/Display/index.js

```diff
@@ -0,0 +1,125 @@
+import React, {useState} from 'react';
+import {View, Text, TextInput, Alert} from 'react-native';
+
+import CalcButton from '../../components/CalcButtons';
+import {styles} from './style';
+
+const Display = () => {
+  const [userText, setUserText] = useState('');
+  const [calcText, setCalcText] = useState('');
+  const handlePress = number => {
+    if (number === '=') {
+      return calculation();
+    }
+    setUserText(userText + number);
+  };
+
+  const calculation = () => {
+    // eslint-disable-next-line no-eval
+    setCalcText(eval(userText));
```

Comment:
Although here makes no difference, just for you to know, usage of `eval` is **strongly** unadvised since it will execute arbitrary code which can lead to very serious security issues.

Reply:
Roger